server/routes/sentencingRoutes.ts

```diff
@@ -0,0 +1,66 @@
+import { RequestHandler } from 'express'
+import type { SentenceLengthForm } from 'forms'
+import trimForm from '../utils/trim'
+import CourtAppearanceService from '../services/courtAppearanceService'
+import { sentenceLengthToSentenceLengthForm } from '../utils/mappingUtils'
+
+export default class SentencingRoutes {
```

Comment:
Take this file with a huge pinch of salt.. still WIP (e.g. the redirects will need changing)